### PR TITLE
Add Pass extension

### PR DESCRIPTION
**Pull request by capipo**
## Description

An extension to handle [the standard unix password manager](https://www.passwordstore.org/) in a convenient way using raycast.

Add `Pass` command that can read passwords on any GPG files in your `~/.password-store` directory, allowing you to copy or automatically paste them into the most-front application.

Highly inspired in [rofi-pass](https://github.com/carnager/rofi-pass) project for linux.

### Showing GPG files on my `~/.password-store` directory.

pass-1

### After selecting the `some-silly-business-site.com` file, its content would be displayed like this.

pass-2

If the user presses enter, the password value will be pasted into the window that was in focus.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/pass/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "categories": [
+    "Security",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Pass",
+      "subtitle": "Find password",
```
Comment (pernielsentikaer) [inline extensions/pass/package.json:17]:
```suggestion
```

File: extensions/pass/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "author": "capipo",
+  "categories": [
+    "Security",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Pass",
```
Comment (pernielsentikaer) [inline extensions/pass/package.json:16]:
```suggestion
      "title": "Search in Pass",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- You're missing an extension icon, you can generate \[one here\](<https://icon.ray.so/>) 
- The extension looks a bit empty when you don’t have pass installed, maybe you should check for that - see how extensions like [Eagle](https://www.raycast.com/Yukai/eagle) is doing it
- Update the REAMe on how to get started

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/pass/package.json (hunk 2 above)
Comment (capipo) [reply extensions/pass/package.json]: Thanks for the comments, it's ready to review again.

**Comment by capipo** [comment]
> Hi 👋
> 
> Thanks for your contribution 💪
> 
> I have now tested your extension, and I have some feedback ready for you:
> 
> * You're missing an extension icon, you can generate [one here](https://icon.ray.so/)
> * The extension looks a bit empty when you don’t have pass installed, maybe you should check for that - see how extensions like [Eagle](https://www.raycast.com/Yukai/eagle) is doing it
> * Update the REAMe on how to get started
> 
> I'm looking forward to testing this extension again 🔥
> 
> Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

Thanks for the comments, it's ready to review again.

File: extensions/pass/package.json (hunk 1 above)
Comment (capipo) [reply extensions/pass/package.json]: 4f7efa35c6dd01a576da8b9b47ef966328670f24

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**